request-server_test.go

```diff
@@ -807,6 +808,25 @@ func TestUncleanDisconnect(t *testing.T) {
 	checkRequestServerAllocator(t, p)
 }
 
+func TestRealPath(t *testing.T) {
+	root := &root{
+		rootFile:       &memFile{name: "/", modtime: time.Now(), isdir: true},
+		files:          make(map[string]*memFile),
+		startDirectory: "/apath",
+	}
+
+	p := root.Realpath(".")
+	require.Equal(t, root.startDirectory, p)
```

Comment:
In the cases where these tests don’t succeed it’s not like anything is going to explode or panic if we continue execution, so `require` might not be entirely necessary.